resource_obmcs_core_console_history.go

```diff
@@ -110,6 +110,11 @@ func (s *ConsoleHistoryResourceCrud) Get() (e error) {
 	return
 }
 
+func (s *ConsoleHistoryResourceCrud) Delete() (e error) {
+	e = nil
```

Comment:
A few lines up in `deleteConsoleHistory` the `return fmt.Errorf("console history resource: console history %v cannot be deleted", d.Id())` line was causing the test to fail even after the test assertion succeeded. It also indicated the resource could not be deleted, this is the pattern for that. Looking at the api docs, there is a DeleteConsoleHistory api--probably added after this was developed a year ago, but I'm more inclined to remove this feature and tests than invest more in it--there is no use case for console history in terraform, it was originally added by mistake.